src/clients/alternatives.rb

```diff
@@ -1,3 +1,3 @@
-require "y2_alternatives/dialog/list_alternatives"
+require "y2_alternatives/client/start_checking_permissions"
 
-Y2Alternatives::Dialog::ListAlternatives.run
+Y2Alternatives::Client::StartCheckingPermissions.new.main
```

Comment:
feels a bit strange about client name, but currently do not have in mind better name .. maybe `Client::Entry` but not sure. so just note, not blocking merge.